tests/test_event.py

```diff
@@ -80,7 +80,7 @@ def test_after_send_error_event_called():
                 service='test',
                 arg1='endpoint',
                 hostport=tchannel.hostport,
-                timeout=0.02,
+                timeout=0.3,
```

Comment:
0.02 timeout is little bit aggressive and make the test flaky. Make it a little bit bigger.